spec/setup/globals.lua

```diff
@@ -1,4 +1,4 @@
-_G.serpent = require("serpent")
+_G.serpent = require("stdlib/utils/serpent")
```

Comment:
While I do like this change, this is a note to self to remove serpent from utils and put it in spec/setup as factorio already includes a custom version of serpent.